example/lib/page_3d_audio.dart

```diff
@@ -17,16 +17,36 @@ class _Page3DAudioState extends State<Page3DAudio> {
 
   SoundProps? currentSound;
   bool spinAround = false;
+  bool canBuild = false;
 
   @override
-  void dispose() {
-    SoLoud.instance.shutdown();
-    SoLoudCapture.instance.stopCapture();
-    super.dispose();
+  void initState() {
+    super.initState();
+
+    /// Ensure the player is down
+    if (SoLoud.instance.isInitialized) {
+      SoLoud.instance.shutdown();
+    }
+
+    /// Reinitialize the player
+    SoLoud.instance.initialize().then((value) {
+      if (value == PlayerErrors.noError) {
+        _log.info('player started');
+        SoLoud.instance.setVisualizationEnabled(false);
+        SoLoud.instance.setGlobalVolume(1);
+        canBuild = true;
+        if (context.mounted) setState(() {});
```

Comment:
Nit, but it's more in style to do it the slightly more verbose way, to signify what state is being set.

```suggestion
        if (context.mounted) {
          setState(() {
            canBuild = true;
          });
        }
```

Reply:
I thought about that, but in this case, if the `build()` has not been called yet (hence the `context.mounted` is false), the `canBuild` bool will not be set to true, and the `build()` will not get called again.
Assuming this is possible!

Reply:
That's not possible. Once `mounted` is `false`, that widget state is out of the picture for good.

```dart
  /// Whether this [State] object is currently in a tree.
  ///
  /// After creating a [State] object and before calling [initState], the
  /// framework "mounts" the [State] object by associating it with a
  /// [BuildContext]. The [State] object remains mounted until the framework
  /// calls [dispose], after which time the framework will never ask the [State]
  /// object to [build] again.
  ///
  /// It is an error to call [setState] unless [mounted] is true.
  bool get mounted => _element != null;
```